frontend/fingertips-frontend/components/forms/SearchForm/index.tsx

```diff
@@ -10,34 +17,57 @@ const StyledInputField = styled(InputField)(
   spacing.withWhiteSpace({ marginBottom: 6 })
 );
 
-export const SearchForm = ({ indicator }: { indicator: string }) => {
-  const initialState: SearchFormState = {
-    indicator: '',
-    message: null,
-    errors: {},
-  };
-  const [_state, formAction] = useActionState(searchIndicator, initialState);
+export const SearchForm = ({
+  searchFormState,
+}: {
+  searchFormState: SearchFormState;
+}) => {
+  const [state, formAction] = useActionState(searchIndicator, searchFormState);
 
   return (
-    <>
-      <H1>Search</H1>
+    <main>
       <form action={formAction}>
+        {state.message && (
+          <ErrorSummary
+            description="At least one of the following fields must be populated:"
+            errors={[
+              {
+                targetName: 'indicator',
+                text: 'Indicator field',
+              },
+            ]}
+            data-testid="search-form-error-summary"
+          />
+        )}
+        <H1>Find public health data</H1>
+        <LeadParagraph>
+          Find public health data by subjects, and health or local authority
+          areas.
+        </LeadParagraph>
+        <InsetText>
+          Use both search options to help you find the most accurate data
+          available.
+        </InsetText>
         <StyledInputField
           input={{
             id: 'indicator',
             name: 'indicator',
-            defaultValue: indicator,
+            defaultValue: searchFormState.indicator,
           }}
-          hint={<>Type in here the IndicatorId.</>}
+          hint={
+            <>
+              For example diabetes, public health indicator, or indicator ID241
```

Comment:
Did you mean to include the `241` here?
```suggestion
              For example diabetes, public health indicator, or indicator ID
```

Reply:
Yes, the ID241 is as it appears in the latest Figma design.

Reply:
I think there's a newer version of the designs (I can see a "round 3") without this. Also, I don't think it makes sense. Even if it was intended as an example it's not formatted in a way that makes it clear.

Reply:
Round 3 looks quite a bit different. Should I be updating with the additional text?

Reply:
We probably need to be careful about spending time tracking the design updates, because we know they're not final and the wording at least will likely change again. I don't think we should worry about it now.

I just think the `241` looks strange and is likely to cause questions when people see it. If you feel really strongly about keeping it I don't want to hold things up, but it just looks like a typo to me.